jina/types/document/__init__.py

```diff
@@ -135,6 +137,10 @@ def __init__(self, document: Optional[DocumentSourceType] = None,
                              f'if you are trying to set the content '
                              f'you may use "Document(content=your_content)"') from ex
 
+        if self._document.id is None or self._document.id == '':
```

Comment:
self._document.id.is_empty() may work better? not sure if works in python

Reply:
Well apparently mine is the best practice in python as far as I know.

Reply:
create a constant on top. MAX_DOCUMENT_ID